stylesheets/_component.block-list.scss

```diff
@@ -207,5 +188,5 @@ $width: 500px;
 }
 
 .block-list--tall .block-list__item {
```

Comment:
Maybe nest the block-list__item inside the .block-list--tall so each object has its own line, as per lines 111, 167 etc.